addon/components/au-control-radio.hbs

```diff
@@ -1,5 +1,5 @@
 <label for="{{@identifier}}" class="au-c-control au-c-control--radio {{this.disabled}}">
-  <input name="{{@name}}" id="{{@identifier}}" value="{{@value}}" disabled={{@disabled}} type="radio" class="au-c-control__input" {{on "change" this.onChange}} ...attributes>
+  <Input class="au-c-control__input" id="{{@identifier}}" name="{{@name}}" disabled={{@disabled}} @type="radio" @checked={{@checked}} {{on "change" this.onChange}} ...attributes />
```

Comment:
you seem to have lost @value here

Reply:
@value is not really needed as it is static in the radio-button case, you can use the regular 'value' (also using @checked & @value in the same input gives you a linting error. From what I learned is that the Ember Input component gives you the power of two-way binding for the '@checked' for example. 

https://github.com/emberjs/ember.js/blob/v3.25.1/packages/%40ember/-internals/glimmer/lib/components/input.ts#L77-L79

Reply:
I tend to view the two-way binding of `@checked` as a legacy feature that will be deprecated sooner or later. the value attribute of a radio button is used to identify which radio button is checked. 

This allows you to use the same callback for all radio buttons for example

Reply:
Yeah I think that might be a better approach, I actually thought that that was the part in radio buttons that needed fixing ? I think it got communicated wrongly then. Thought there was need for a way to get both the checked state & its value after an action

Reply:
given the discussion in our #ember channel and on https://discuss.emberjs.com/t/ember-octane-and-2-way-binding-what-are-you-recommend/18092/15 I'd avoid wrapping the `Input` component and rather use the native `input` element.

Reply:
but perhaps we should discuss the intend of this PR first :)